### DISCUSSION TITLE
Disabling "Visual Mode" just flashbang the user

### DISCUSSION THREAD

**Discussion by Scarlaid** [seerge/g-helper#3524] [Q&A] [answered]
I have been on Default config of Visual Mode for all time because it's literally how Armory Crate set it up, since G-Helper now introduce an option to completely disable it, and I clearly see why as the color looks more accurate now

but I found out that restarting, or just turning the laptop off then turn it on, with `Disabled` give out a color bug
Heres the video: https://streamable.com/tqubu0

**Comment by seerge** [comment]
@Scarlaid hello,

I believe you :) But since all Visual Modes (Game Visual in AC) are actually handled by AsusSplendid.exe utility under the hood, including Disabled, it may be just incompatible with AMD drivers from AMD itself. As when you use ones from Asus website directly - it works fine.

I would say - just use Default instead ? :)

Thanks

**Reply by Scarlaid** [reply]
bummer, so it's just working as intended?

I don't mind using Default, I just recently find that by disabling the Visual Mode, the color just looks, better? I don't know, my eyes could be decieving me, but it does just give the colors more constrats

video: https://streamable.com/5f6fkl

**Reply by seerge** [reply, marked as answer]
@Scarlaid no, it's not working "as intended". It's just asus didn't fix it still by some reason or tested only with their own AMD drivers. Hopefully eventually they will fix it.
